config_examples/filtering_creditcards.yaml

```diff
@@ -0,0 +1,37 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+
+processors:
+  redaction:
+    allow_all_keys: true
+    allowed_keys:
+      - credit_card
```

Comment:
Would this also work without these two lines since we already have `allow_all_keys: true` which should implicitly allow `credit_card` and only blocked `blocked_values`?

From https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/redactionprocessor/README.md:

    # allow_all_keys is a flag which when set to true, which can disables the
    # allowed_keys list. The list of blocked_values is applied regardless. If
    # you just want to block values, set this to true.
    allow_all_keys: false
    # allowed_keys is a list of span/log/datapoint attribute keys that are kept on the span/log/datapoint and
    # processed. The list is designed to fail closed. If allowed_keys is empty,
    # no attributes are allowed and all span attributes are removed. To
    # allow all keys, set allow_all_keys to true.
    allowed_keys:

Reply:
oh good catch!, my bad, there should be `allow_all_keys: false`. Will adapt it in a follow up PR

Reply:
--> #340